app/components/lg-image-gallery.js

```diff
@@ -9,6 +9,13 @@ export default Ember.Component.extend({
   isVisible: Ember.computed.gt('imageSet.images.length', 0),
 
   actions: {
+    saveImageSet: function() {
+      var imageSet = this.get('imageSet');
+      if (imageSet.get('id')) {
+        imageSet.save();
```

Comment:
in progress and error messages will show up in the template while this is saving.